app/src/main/java/mn/devfest/api/model/Session.java

```diff
@@ -102,6 +113,11 @@ public String getCategory() {
     }
 
     public DateTime getStartTime() {
-        return startTime;
+        try {
+            return DateTime.parse(startTime);
+        } catch (Exception e) {
+            Timber.e(e, "Failed to parse the startTime");
+            return new DateTime();
+        }
```

Comment:
Ugh. So, I originally did this because Firebase is returning strings, and I wasn't sure how to get that to map to DateTimes with getValue(). I tried changing the setter, but that doesn't seem to be working. This will probably take a few more days to fix, and I'm starting to get really concerned about my ability to get the app done before the conference.